fasta/pFasta/tools.go

```diff
@@ -92,3 +48,67 @@ func Sample(input []PFasta, chrom string) fasta.Fasta {
 
 	return answer
 }
+
+// faToPfa returns a pFasta representation of the given Fasta sequence, start inclusive, end exclusive
+func faToPfa(input fasta.Fasta, start int, end int) PFasta {
+	if end == -1 {
+		end = len(input.Seq)
+	} else if end > len(input.Seq) {
+		log.Fatalf("Requested end argument (%v) out of range.", end)
+	}
+
+	answer := PFasta{Name: input.Name, Seq: make([]pDna.Float32Base, end-start)}
+
+	fasta.ToUpper(input)
+	var base dna.Base
+	var idx int
+	for idx, base = range input.Seq[start:end] {
+		if base == dna.A {
+			answer.Seq[idx] = pDna.Float32Base{A: 1, C: 0, G: 0, T: 0}
+		} else if base == dna.C {
+			answer.Seq[idx] = pDna.Float32Base{A: 0, C: 1, G: 0, T: 0}
+		} else if base == dna.G {
+			answer.Seq[idx] = pDna.Float32Base{A: 0, C: 0, G: 1, T: 0}
+		} else if base == dna.T {
+			answer.Seq[idx] = pDna.Float32Base{A: 0, C: 0, G: 0, T: 1}
+		} else if base == dna.N {
+			answer.Seq[idx] = pDna.Float32Base{A: 0, C: 0, G: 0, T: 0}
```

Comment:
Not saying this is a problem, but I'm curious why this is all zeros for N as opposed to 0.25 for everything? Was there an experimental reason or a theoretical reason?